### DISCUSSION TITLE
How to change keys?

### DISCUSSION THREAD

**Discussion by strbrgr** [manna-harbour/miryoku#172] [Q&A] [answered]
I recently got a Keebio Iris Rev7 and did flash the Miryoky_qmk layout on it via 
```qmk flash -c -kb keebio/iris/rev7 -km manna-harbour_miryoku```

Is there a way to use the qmk CLI to create the miryoku keymap before it gets flashed as hex file to my keyboard? I would like to make my own custom changes but was not able to figure out how to create the keymap first.

**Comment by manna-harbour** [comment, marked as answer]
The QMK source files are all there, it's just more complicated than a regular keymap. See https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku.

To customise, see https://github.com/manna-harbour/miryoku/discussions/85.

To choose from the built-in alternative layout options, see https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#qmk and https://github.com/manna-harbour/miryoku/tree/master/docs/reference#alternative-layouts.

**Reply by strbrgr** [reply]
Thank you manna-harbour, I was able to remap keys. After doing that, how am I able to create my keymap.c? I would like to add combo keys to my setup.
